src/Microsoft.Azure.SignalR.Common/Microsoft.Azure.SignalR.Common.csproj

```diff
@@ -26,4 +26,7 @@
     <PackageReference Include="Microsoft.AspNetCore.Connections.Abstractions" Version="$(MicrosoftAspNetCoreConnectionsAbstractionsPackageVersion)" />
     <PackageReference Include="Microsoft.Extensions.Logging.Abstractions" Version="$(MicrosoftExtensionsLoggingAbstractionsPackageVersion)" />
   </ItemGroup>
+  <ItemGroup>
+    <Folder Include="ServiceConnections\scopes\" />

```

Comment:
what is this folder for?

Reply:
updated. async local scopes are stored here.